src/quo/components/wallet/required_tokens/view.cljs

```diff
@@ -0,0 +1,66 @@
+(ns quo.components.wallet.required-tokens.view
+  (:require [quo.components.markdown.text :as text]
+            [quo.components.utilities.token.view :as token]
+            [quo.components.wallet.required-tokens.style :as style]
+            quo.theme
+            [react-native.core :as rn]
+            [schema.core :as schema]))
+
+(def ?schema
```

Comment:
🙌

Reply:
@clauxx, @yqrashawn - can you take a look at the schema :)

Reply:
This is my first attempt at crafting a schema, and I welcome any feedback or suggestions!

Reply:
do we have any docs about usage of schema? and if we define schema do we still need to use docstring for components?

Reply:
Good point @mohsen-ghafouri - I think we should not use doc strings if using schemas

Reply:
That's a good point. I believe we can use the schema alone, which reduces maintenance overhead. IMO schemas are generally easier to keep up-to-date compared to docstrings.

Reply:
schema below LGTM